catalog/task/push-sbom-to-pyxis/0.1/README.md

```diff
@@ -19,6 +23,10 @@ tasks:
     taskRef:
       name: push-sbom-to-pyxis
     params:
-      - name: imageURL
-        value: 'quay.io/my-org/my-repo:latest'
+      - name: imageURLs
```

Comment:
The example usage is no longer correct now that we use bundle resolvers. Can you either fix it or drop it from the README (seems other tasks don't have one, although it is quite nice)

Reply:
Thanks, removed. The example section definitely wasn't my invention, not sure where I copied it from. But it's kind of a duplicate, since the same thing is in the sample yaml.